### DISCUSSION TITLE
It's a display that won't go away with all the solutions on the web, but it's a bug anyway, so just tell me if I can leave it alone.

### DISCUSSION THREAD

**Discussion by kendji** [browserslist/browserslist#773] [General]
amplify blahblah
.
.
.
Browserslist: caniuse-lite is outdated. Please run:                                                                 <- i wanna remove here
  npx update-browserslist-db@latest                                                                                    <- and here
  Why you should do it regularly: https://github.com/browserslist/update-db#readme      <- and here

I've updated it a million times with npm and yarn.

**Comment by kendji** [comment]
Sorry, is it an Amplify issue?

**Comment by ai** [comment]
Check your lock file (`package-lock.json` or `yarn.lock`) for `caniuse-lite`. If you have it, update to the latest version (and do it regularly, it will really affect the performance).

If you don’t have it, it could be an issue of Amplify if they bundle it into their files without a way to update.
